Atomic/run.py

```diff
@@ -38,6 +39,17 @@ def run(self):
             args = self._get_args("RUN")
             if args:
                 args += self.command
+                opts_file = self._get_args("RUN_OPTS_FILE")
+                if opts_file:
+                    opts_file = self.sub_env_strings("".join(opts_file))
+                    if opts_file.startswith("/"):
+                        try:
+                            run_opts = open(opts_file, "r").read()
+                            os.environ["RUN_OPTS"] = run_opts
```

Comment:
Bisecting the history, it was already 0cba782a03d110a154550fa5f1ad5a9e7cfacbd5 which introduced the regression.